### DISCUSSION TITLE
Storybook 8 compatibility

### DISCUSSION THREAD

**Discussion by joevaugh4n** [chakra-ui/chakra-ui#8306] [General]
Hey, Chakra UI! I’m reaching out from the Storybook core team :)

Storybook 8 (our next major release) is coming soon, and it introduces some breaking changes that your addon might need to account for.

We’ve listed all the key changes in [a new Storybook 8 migration guide](https://storybook.js.org/docs/8.0/addons/addon-migration-guide#key-changes-for-addons), and we’d encourage you to check it out and see if the Chakra UI Storybook addon is impacted.

We also have an addons channel in our Discord community, which we’d like to use to better communicate with addon authors in the future. If you’re not part of the server, consider joining [here](https://discord.gg/wMff8FkF) and message me (@joevaugh4n) so that I can give you the addon creator role! If you’re already in the server, please just ping me and I’ll respond to you there.

Thanks for being a part of our addon community!

Joe

**Comment by jacklim-upfrnt** [comment]
It seems like it is incompatible with Storybook 8 based on my own local testing. Could anyone confirm?

**Comment by segunadebayo** [comment]
Storybook works. See https://chakra-ui.com/docs/get-started/frameworks/storybook
